comp/core/tagger/origindetection/origindetection.go

```diff
@@ -0,0 +1,75 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I think we should add somewhere in this file that part of it (constants, struct) is currently duplicated.
If the refactor is not so straightforward it can be left for a future PR, but we should track it.

Reply:
Done 👍